### ISSUE TITLE
Property '"options"' is incompatible with index signature

### ISSUE THREAD

**Issue by skrappyzz** [Grsmto/simplebar#702]
**Describe the bug**
When import { SimplebarAngularModule } from "simplebar-angular"

**To Reproduce**
1.Install simplebar-angular by using "npm install simplebar-angular --save"
2. Add import to app.module.ts as described in the simplebar-angular documentation ( import { SimplebarAngularModule } from 'simplebar-angular'
4. See error in cli: Error: node_modules/simplebar-angular/lib/simplebar-angular.component.d.ts:16:95 - error TS2344: Type '{ options: { alias: "options"; required: false; }; }' does not satisfy the constraint '{ [key: string]: string; }'.       
  Property '"options"' is incompatible with index signature.
    Type '{ alias: "options"; required: false; }' is not assignable to type 'string'.

static ɵcmp: i0.ɵɵComponentDeclaration<SimplebarAngularComponent, "ngx-simplebar", never, { "options": { "alias": "options"; "required": false; }; }, {}, never, ["*"], false, never>;

**Expected behavior**

**Environment**

| Software         | Version(s) |
| ---------------- | ---------- |
| SimpleBar        | 3.2.5 and 3.2.6
| Browser          | Chrome
| npm        | 10.7.0
| Operating System | Windows 10

**Comment by letovsk1y** [comment]
I have the same issue, but it can be fixed by installing simplebar-angular 3.2.4 version

**Comment by skrappyzz** [comment]
> I have the same issue, but it can be fixed by installing simplebar-angular 3.2.4 version

Yep, I mentioned it in the description that affected versions are 3.2.5 and 3.2.6

**Comment by Grsmto** [comment]
There's zero change in types or any angular code between 3.2.4 and 3.2.5 so I have absolutely no clue why this would happening. 
Also I tried running the angular example in the repo but couldn't get it to work for some reason, spent a good 45min on it. If someone has any idea why this typing error magically happened I would be happy to fix it.

**Comment by skrappyzz** [comment]
Hey @Grsmto, I just compared versions 3.2.4 and 3.2.5 and noticed this change:  https://github.com/Grsmto/simplebar/commit/9a2cdc9f444f362a838fb894d5bbcb287cdca14c#diff-874e3e4b59bff724c2ffc471287446a81ee8296ae357129c8e06032cd2b07939R9 Could this be the source of the issue?

**Comment by Grsmto** [comment]
I don't see how, this is the vanilla js package, the angular plugin does not have a dependency on this file...

**Comment by Haves1001** [comment]
I actually have this issue also appearing when using 3.2.4 up till 3.0.0 with Angular 16 - so maybe a change introduced earlier?

**Comment by skrappyzz** [comment]
@Grsmto this issue happens with Angular 15, in Angular 18 it works fine. Any ideas?
